src/app/ngrx/reducers/episode.reducer.ts

```diff
@@ -1,39 +1,38 @@
-import { ActionTypes, ActionWithPayload, CmsEpisodeGuidPayload } from '../actions';
+import { ActionTypes, ActionWithPayload, CmsEpisodeGuidsPayload, CmsAllPodcastEpisodeGuidsAction  } from '../actions';
 import { EpisodeModel } from '../model';
 
 const initialState = [];
 
 // TODO: should be able to use ActionWithPayload<All> here with the union type,
```

Comment:
This All type is still garbage, and I don't know why. It hasn't actually mattered yet though, and instanceof is still a work around if it ever does.